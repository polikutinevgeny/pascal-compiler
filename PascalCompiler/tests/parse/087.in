const
    a = 10.1;
    b = 10;
    c = 'c';
begin
end.