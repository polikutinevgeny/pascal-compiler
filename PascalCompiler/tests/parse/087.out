program 
  integer;
  real;
  char;
  a: real = 10.1;
  b: integer = 10;
  c: char = c;
